modules/distr-tracing-tempo-config-default.adoc

```diff
@@ -82,7 +82,7 @@ spec:
 |
 
 |`storage:`
-|Configuration options that define the storage. All storage-related options must be placed under `storage` and not under the `allInOne` or other component options.
+|Configuration options that define the storage.
```

Comment:
Not really necessary, and really more of a nit, but is there a noun we could use after storage? storage class? storage provider? storage component?

Reply:
I think _storage provider_ would work in this context.

Reply:
Thank you both. The current text quotes the `storage` parameter, so it is what it is.